packages/ember-application/lib/system/application-instance.js

```diff
@@ -165,13 +163,7 @@ const ApplicationInstance = EngineInstance.extend({
       if (options.isInteractive) {
         this.setupEventDispatcher();
       }
```

Comment:
Can you fix the indentation here? After removing the guard, there's now an extra level of indentation here